.github/workflows/nexus-publish-snapshots.yml

```diff
@@ -10,7 +10,7 @@ on:
       - development
 
 jobs:
-  build:
+  publish_snapshot:
```

Comment:
NIT: we named this `publish_snapshots` in sync with the GitHub action name. I am okay with both

Reply:
But its technically wrong, it deploys only ONE snapshot :)

Reply:
yes :) the action files name contains the plural tough 🤷‍♂️ As we do not actively use this I think it is fine

Reply:
yes :) the action files name contains the plural tough 🤷‍♂️ As we do not actively use this I think it is fine